addons/web/static/src/legacy/scss/primary_variables.scss

```diff
@@ -156,3 +156,7 @@ $o-kanban-dashboard-dropdown-complex-gap: 5px !default;
 // Form view
 
 $o-form-view-sheet-max-width: 1140px !default;
+
+// Notifications
+$o-notification-width: 320px !default;
```

Comment:
`$o-notification-width` should not be necessary (we can use boostrap default value for `toast` classes)
`$o-notification-border-width`: use a value proportional to the default border width  (`1px`), eg. `5px`


From now on move variables in a new file, in your case `addons/web/static/src/core/notifications/notification.variables.scss`.

Chances are that your PR will be merged before mine,  so add this line your web's manifest
https://github.com/odoo/odoo/pull/82434/files#diff-66d417d1828879bbc2124ddc118b47030ea7c30fb93e660c5231365420002214R304